.circleci/config.yml

```diff
@@ -8,30 +8,39 @@ jobs:
       - checkout
       - setup_remote_docker
       - run:
-          name: Install build packages
-          command: apk --no-cache add bash pcre-tools
+          name: Install packages
+          command: apk --no-cache add bash curl pcre-tools
       - run:
           name: Build docker images
           command: |
             ./build.sh
       - run:
           name: Push docker images
+          shell: /bin/bash -Eeuo pipefail
           command: |
+            does_repo_exist() {
+              curl --silent -f -lSL https://index.docker.io/v1/repositories/scalableminds/$1/tags > /dev/null
+            }
+
             echo $DOCKER_PASS | docker login -u $DOCKER_USER --password-stdin
             for APP in $(./get_valid_order.sh); do
-              docker push scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM} || {
+              if does_repo_exist $APP; then
+                docker push scalableminds/$APP:${CIRCLE_BRANCH}__${CIRCLE_BUILD_NUM}
+                docker push scalableminds/$APP:${CIRCLE_BRANCH}
+                eval "MICROBADGER_HOOK=\$MICROBADGER_HOOK_$APP"
```

Comment:
Where does `$MICROBADGER_HOOK_$APP` come from?

Reply:
this evals e.g. to `MICROBADGER_HOOK=$MICROBADGER_HOOK_cronut`
This comes from the CI env vars:
https://circleci.com/gh/scalableminds/dockerfiles/edit#env-vars